go.mod

```diff
@@ -266,7 +266,7 @@ replace github.com/textileio/go-ds-badger3 => github.com/anyproto/go-ds-badger3
 
 replace github.com/dgraph-io/ristretto => github.com/dgraph-io/ristretto v0.1.1-0.20220403145359-8e850b710d6d
 
-replace github.com/JohannesKaufmann/html-to-markdown => github.com/anyproto/html-to-markdown v0.0.0-20230814233030-a0434e69902f
+replace github.com/JohannesKaufmann/html-to-markdown => github.com/anyproto/html-to-markdown v0.0.0-20230919201515-d902dc00305a
```

Comment:
https://github.com/anyproto/html-to-markdown/commit/a62da379cb894253e90b1fe06c0dd079452f97ce